src/sentry/static/sentry/app/components/progressBar.tsx

```diff
@@ -1,17 +1,58 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
+import {Theme} from 'app/utils/theme';
+
+type Variant = 'small' | 'large';
+
 type Props = {
   /**
    * The value of the progress indicator for the determinate variant. Value between 0 and 100
    */
   value: number;
+  /**
+   * The style of the progressBar
+   */
+  variant?: Variant;
   /**
    * Styles applied to the component's root
    */
   className?: string;
 };
 
+const getVariantStyle = ({
+  variant = 'small',
+  theme,
+}: Pick<Props, 'variant'> & {theme: Theme}) => {
+  if (variant === 'large') {
+    return `
+      height: 24px;
+      border-radius: 24px;
+      border: 1px solid ${theme.gray200};
```

Comment:
Should we use the border alias from the theme instead?

Reply:
we don't have a border-radius with 24px in our theme, but in case we use it more often I can add it there 😉

Reply:
oh, I meant the color:
https://github.com/getsentry/sentry/blob/5abd9bb42cd14efcd6a12391348a0d2495fd619d/src%2Fsentry%2Fstatic%2Fsentry%2Fapp%2Futils%2Ftheme.tsx#L82-L85